tests/app/main/test_views.py

```diff
@@ -6,7 +6,7 @@
     from io import BytesIO as StringIO
 
 import mock
-from dmutils.apiclient import APIError
+from dmutils.apiclient import HTTPError, REQUEST_ERROR_MESSAGE
```

Comment:
That's a good point, but I think it applies to the dmutils tests more than it applies here. If we decide to change the default error message I think it shouldn't make the _app_ tests fail.